code/modules/detectivework/tools/sample_kits/_sample.dm

```diff
@@ -0,0 +1,56 @@
+
+/obj/item/forensics/sample
+	name = "forensic sample"
+	icon = 'icons/obj/forensics.dmi'
+	var/object
+	var/list/possible_evidence_types = list(/datum/forensics/fibers)
+	var/list/evidence
+
+/obj/item/forensics/sample/Initialize(mapload, var/atom/supplied)
+	. = ..(mapload)
+	if(supplied)
+		copy_evidence(supplied)
+		name = "[initial(name)] (\the [supplied])"
+		object = "[supplied], at [get_area(supplied)]"
+	update_icon()
+
+/obj/item/forensics/sample/Destroy()
+	. = ..()
+	QDEL_NULL_LIST(evidence)
```

Comment:
last I checked this runtimes on non-instance types, so if this has strings and instances, you need to not qdel the strings.

Reply:
nope here it'll be `/datum/forensics` datums so should be fine